mysql-test/suite/client/client-env-variable.test

```diff
@@ -0,0 +1,64 @@
+-- source include/have_log_bin.inc
+
+# Set up environment varibles for client programs
+--let MYSQLADMIN = $MYSQL_BINDIR/client//mariadb-admin
+--let MYSQLBINLOG = $MYSQL_BINDIR/client//mariadb-binlog
+--let MYSQLCHECK = $MYSQL_BINDIR/client//mariadb-check
+--let MYSQLDUMP = $MYSQL_BINDIR/client//mariadb-dump
+--let MYSQLIMPORT = $MYSQL_BINDIR/client//mariadb-import
+--let MYSQLSHOW = $MYSQL_BINDIR/client//mariadb-show
+--let MYSQLSLAP = $MYSQL_BINDIR/client//mariadb-slap
```

Comment:
These are all set in mysql-test/mariadb-test-run.pl, though usually as with underscore like `$MYSQL_CHECK`.

Reply:
The environment variables for the client programs have a default options file argument in them. The options file was  stopping me from testing the client programs

Reply:
In that case, they should probably be things like  `MARIABINLOG`, etc...